src/utilities/getRandom.js

```diff
@@ -0,0 +1,3 @@
+export function getRandom(number) {
```

Comment:
Change the name function so that it's more obvious that it generates a number between 0 and `number -1`